arch/x86/core/intel64/cpu.c

```diff
@@ -90,6 +90,15 @@ struct x86_tss64 tss3 = {
 };
 #endif
 
+/* We must initialize with something OTHER THAN 0, or else it will land into
+ * .bss: in this case, though locore.S initalizes it relevantly, all will be
```

Comment:
Might consider using an explicit section directive here instead of relying on the side effect of a noop initialization.

Reply:
That would indeed be clearer.